templates/settings.py.erb

```diff
@@ -9,7 +9,7 @@ DATABASES = {
         'PASSWORD': '<%= scope['pulpcore::postgresql_db_password'] %>',
         'HOST': '<%= scope['pulpcore::postgresql_db_host'] %>',
         'PORT': '<%= scope['pulpcore::postgresql_db_port'] %>',
-<% if scope['pulpcore::postgresql_db_ssl'] && !scope['pulpcore::postgresql_db_manage'] -%>
+<% if scope['pulpcore::postgresql_db_ssl'] && !scope['pulpcore::postgresql_manage_db'] -%>
```

Comment:
Looks great, @rafaelguerra01 ! Thanks for fixing my bug

Reply:
This indicates a lack of test coverage. It would be good to add a test for this.